diff --git a/node_modules/apollo-cache-persist/Cache.d.ts b/node_modules/apollo-cache-persist/Cache.d.ts
index 9387aa7..4f505ae 100644
--- a/node_modules/apollo-cache-persist/Cache.d.ts
+++ b/node_modules/apollo-cache-persist/Cache.d.ts
@@ -1,4 +1,4 @@
-import { ApolloCache } from 'apollo-cache';
+import { ApolloCache } from '@apollo/client/core';
 import { ApolloPersistOptions, PersistedData } from './types';
 export default class Cache<T> {
     cache: ApolloCache<T>;
diff --git a/node_modules/apollo-cache-persist/types/index.d.ts b/node_modules/apollo-cache-persist/types/index.d.ts
index 9d4f55f..606f7b2 100644
--- a/node_modules/apollo-cache-persist/types/index.d.ts
+++ b/node_modules/apollo-cache-persist/types/index.d.ts
@@ -1,4 +1,4 @@
-import { ApolloCache } from 'apollo-cache';
+import { ApolloCache } from '@apollo/client/core';
 export declare type LogLevel = 'log' | 'warn' | 'error';
 export declare type LogLine = [LogLevel, any[]];
 export declare type TriggerUninstallFunction = () => void;
